mesonbuild/backend/ninjabackend.py

```diff
@@ -2234,10 +2234,17 @@ def generate_cython_compile_rules(self, compiler: 'Compiler') -> None:
         description = 'Compiling Cython source $in'
         command = compiler.get_exelist()
 
-        args = ['$ARGS', '$in']
+        depargs = compiler.get_dependency_gen_args('$out', '$DEPFILE')
+        if depargs:
+            depfile = '$out.dep'
+        else:
+            depfile = None
```

Comment:
I would write that as
```python
depfile = '$out.dep' if depargs else None
```
as I find it more readable this way.

Reply:
Agreed, and LGTM otherwise.

Reply:
Done.